src/test/http/app.test.ts

```diff
@@ -322,6 +325,97 @@ describe("server", () => {
         expect(resp.statusCode).toBe(200);
         expect(resp.body).toEqual(expectedResults);
       });
+
+      test.only("should estimate matching with new votes for projects", async () => {
```

Comment:
The `only` here looks like a leftover. (Another good candidate for the linter, incidentally.)

Reply:
Thanks - fixed and added an eslint plugin and rule to forbid `only`s